### ISSUE TITLE
Missing form label - Create Plan

### ISSUE THREAD

**Issue by mariapraetzellis** [CDLUC3/dmptool#368]
On: https://dmptool.org/plans/new

CODE EXAMPLE:
<select name="plan[template_id]" id="plan_template_id" class="form-control" aria-describedby="template-selection"></select>

Applicable WCAG Standard(s)
1.1.1 Non-text Content (Level A)
1.3.1 Info and Relationships (Level A)
2.4.6 Headings and Labels (Level AA)
3.3.2 Labels or Instructions (Level A)
